uavcan_data_types/cvra/odometry/20160.WheelEncoder.uavcan

```diff
@@ -0,0 +1,6 @@
+#
+# Differential base Encoder Position
+#
+
+uint16 left_encoder_raw
```

Comment:
I think it would make more sense to have them as 32 bit values, 16 bit will overflow quite early. We can also choose to handle the overflow, but it is just added complexity.

Reply:
Depends if the timers support 32bit. Also, we're sending with 100Hz, I don't think this will overflow between two messages.

Reply:
Looking at the `motor-control-firmware` code, it seems that the QEI support 32 bits.

I don't think we should base our API definitions on the hardware. I would rather have the overflow handled locally, and the non-overflowing value sent over UAVCAN, in order to have the board always send a value that make sense.

Reply:
So yesterday we discussed this. The idea that we reached was the following: loosing one CAN frame should not be an issue. So our implementation publishes a processed `int32` that contains the already-integrated value.